### ISSUE TITLE
train target is not local affine transformation as in paper?

### ISSUE THREAD

**Issue by nopattern** [sergiomsilva/alpr-unconstrained#20]
Thank you for your great job
      I find the model output is 
        `xprobs    = Conv2D(2, 3, activation='softmax', padding='same')(x)
	xbbox     = Conv2D(6, 3, activation='linear' , padding='same')(x)`
      which has 2+6 = 8 parameters as in paper.

    But  in your augment sampling  code 
     `		for k in range(batch_size):
			data = choice(Data)
			XX,llp,pts = augment_sample(data[0],data[1].pts,dim)
			YY = labels2output_map(llp,pts,dim,model_stride)

			Xtrain[k] = XX
			Ytrain[k] = YY`

    The YY label is 1 class label and 8 points ,  so the model output and label mismatch.
      Do I misunderstand something?

**Comment by nopattern** [comment]
Ok, I have found the loss.py in src directory which solve my   confuse.
